src/main/java/dota/buff/service/impl/PlayerServiceImpl.java

```diff
@@ -16,13 +16,13 @@
 import java.util.stream.Collectors;
 
 @Service
-public class UserServiceImpl implements UserService {
+public class PlayerServiceImpl implements PlayerService {
```

Comment:
You are able to use @RequiredArgsConstructor for this and other services as well

Reply:
Ok